src/tooltip/Tooltip.tsx

```diff
@@ -44,7 +44,7 @@ const defaultProps = {
   highlightColor: 'transparent',
   withPointer: true,
   toggleOnPress: true,
-  toggleAction: 'onPress',
+  toggleAction: 'onPress' as const,
```

Comment:
I'm not sure I understand the necessity of this change.

Reply:
That's necassary! it's just new syntax that needs to be written in order to do that! 
Actually that wasn't previously written and the error which might arise due to that was hidden because of ts-ignore.Basically that's just a syntax!